### PR TITLE
fix(ext/node): process.getBuiltinModule

### PR DESCRIPTION
**Pull request by littledivy**
Closes https://github.com/denoland/deno/issues/26832

### CODE REVIEW

**Review by bartlomieju — APPROVED** [review]

File: ext/node/polyfills/01_require.js [hunk 1]
```diff
@@ -1233,6 +1233,26 @@ function isBuiltin(moduleName) {
...
+
+  const mod = loadNativeModule(id, id);
+  if (mod) {
+    return mod.exports;
+  }
+
+  return undefined;
+}
+
+Module.getBuiltinModule = getBuiltinModule;
```
Comment (nicolo-ribaudo) [inline ext/node/polyfills/01_require.js:1254]: `Module.getBuiltinModule` does not exist in Node.js

File: tests/unit_node/process_test.ts [hunk 2]
```diff
@@ -1152,3 +1152,8 @@ Deno.test("process.stdout.columns writable", () => {
   process.stdout.columns = 80;
   assertEquals(process.stdout.columns, 80);
 });
+
+Deno.test("getBuiltinModule", () => {
+  assert(process.getBuiltinModule("fs"));
+  assertEquals(process.getBuiltinModule("something"), undefined);
```
Comment (bartlomieju) [inline tests/unit_node/process_test.ts:1158]: Actually please add a test that uses `node:fs` too

**Review by bartlomieju — CHANGES_REQUESTED** [review]
https://github.com/denoland/deno/pull/26833/files#r1838268827

**Comment by nicolo-ribaudo** [comment]
Thank you! :)

**Pull request merged**